services/loadbalancer/api_default.go

```diff
@@ -26,6 +26,123 @@ import (
 // DefaultApiService DefaultApi service
 type DefaultApiService service
 
+type ApiAPIServiceGetQuotaRequest struct {
```

Comment:
It seems there is a new onboarded method to the SDK, but the LoadBalancer didnt beatutified it :cry: 

Can you skip the LoadBalancer in this PR so that we can still ask them to fix it?